aiida/cmdline/commands/cmd_profile.py

```diff
@@ -133,69 +135,49 @@ def profile_setdefault(profile):
 
 
 @verdi_profile.command('delete')
-@options.FORCE(help='to skip questions and warnings about loss of data')
-@click.option(
-    '--include-config/--skip-config',
```

Comment:
I agree with removing this option, but what was the use case of `--skip-config`?

Reply:
Joost mag het weten. Honestly, I have no clue, but it was already deprecated and I certainly don't see the point in supporting it.

Reply:
Me neither. 😅 De vuilbak in!